athena_cli.py

```diff
@@ -134,6 +158,7 @@ def do_help(self, arg):
         help_output = """
 Supported commands:
 QUIT
+EXIT
```

Comment:
Do `EXIT` and `QUIT` do exactly the same thing?

Reply:
yes, "exit" is supported by other CLIs (like the presto-cli) so some users liked have it

Reply:
You're right. https://github.com/prestodb/presto/blob/master/presto-cli/src/main/java/com/facebook/presto/cli/Console.java#L229